### DISCUSSION TITLE
Is it possible to change which camera an image is associated with in Frigate+?

### DISCUSSION THREAD

**Discussion by pauln** [blakeblackshear/frigate#4931] [Ask A Question] [answered]
I realised after uploading a batch of images that I'd forgotten to select the correct camera, so now they're associated with the wrong one (different height/angle on a different scene - driveway vs back garden).  I can't see a way to change which camera the images are associated with - or to remove them in order to reupload them against the correct camera - so I'm guessing this hasn't been implemented yet; is there a preferred way to suggest features for Frigate+ such as these?

**Comment by blakeblackshear** [comment, marked as answer]
Unfortunately, there isn't a way to change this yet. How many images are we talking about? Does the camera have any images that are correct, or are they all wrong?

**Reply by pauln** [reply]
It's 15 images; I haven't processed them (marking objects and verifying/saving) since I realised shortly after hitting the upload button, and I've refrained from uploading any further images so they're the most recent images uploaded to both that camera and my account.  Both cameras (the one the images are actually from and the one they've been incorrectly assigned to) have a number of correctly-assigned images which I've already processed.  I imagine you're probably too busy preparing for the 0.12 release (which looks like it'll be a great update!) to add new features to the Frigate+ UI at the moment - should I drop you an email with the details for you to reassign (or delete) the images this time?

When you do get a chance to work on Frigate+, I'd suggest defaulting the camera dropdown on the manual upload page to be blank / a placeholder, preventing upload until a valid camera is chosen (though if only one camera is configured, you could still default to it for convenience).  Alternatively, you could shift the "add image" button to a camera-specific page and use that context to pick the default camera.  That should reduce the chances of this happening again - I'm sure I won't be the only person to forget to change from the "default" camera when uploading ;-)

**Reply by blakeblackshear** [reply]
Those are good ideas. Thanks for the feedback. If you want, I can probably manually update them in a query if you send me the image IDs for those 15 at blake@frigate.video. You can find it when clicking the image thumbnail to go into the editor in the URL.

**Reply by pauln** [reply]
Thanks - I've chucked you an email with the list of IDs.
